docs/infrastructure.md

```diff
@@ -1,37 +1,79 @@
 # SysAdmin
 
-These instructions are for a sysadmin looking to deploy Neurobagel locally in an institute or lab.
+These instructions are for a sysadmin looking to deploy Neurobagel locally in an institute or lab. 
+A local neurobagel deployment includes the neurobagel API, 
+and a graph backend to store the harmonized metadata.
```

Comment:
Doesn't a local deployment also include the query tool now?

Reply:
Yeah, good point. It's part of the docker-compose. So I'll mention it. Thanks!